packages/arb-token-bridge-ui/src/hooks/useTransactions.ts

```diff
@@ -416,44 +364,6 @@ const useTransactions = (): [Transaction[], TransactionActions] => {
     })
   }
 
-  const fetchAndUpdateL1ToL2MsgClassicStatus = async (
-    txID: string,
-    l1ToL2Msg: L1ToL2MessageReaderClassic,
-    isEthDeposit: boolean,
-    status: L1ToL2MessageStatus
-  ) => {
-    const isCompletedEthDeposit =
-      isEthDeposit && status >= L1ToL2MessageStatus.FUNDS_DEPOSITED_ON_L2
-
-    const l2TxID = (() => {
-      if (isCompletedEthDeposit) {
-        return l1ToL2Msg.retryableCreationId
-      }
-
-      if (status === L1ToL2MessageStatus.REDEEMED) {
-        return l1ToL2Msg.l2TxHash
-      }
-
-      return undefined
-    })()
-
-    updateTxnL1ToL2MsgData(txID, {
-      status: isCompletedEthDeposit
-        ? L1ToL2MessageStatus.FUNDS_DEPOSITED_ON_L2
-        : status,
-      l2TxID,
-      fetchingUpdate: false,
-      retryableCreationTxID: l1ToL2Msg.retryableCreationId
-    })
-  }
-
-  const removeTransaction = (txID: string) => {
-    return dispatch({
-      type: 'REMOVE_TRANSACTION',
-      txID: txID
-    })
-  }
-
   const setTransactionSuccess = (txID: string) => {
```

Comment:
Don't think it's used anymore either

Reply:
It's being used in `updateTransaction` which in-turn is being used in the UI code